documentation/src/main/wiki/GettingStarted.md

```diff
@@ -260,3 +264,32 @@ This section configures the Services mock used by the example application. It co
 
 Please mind that this an example that depicts a valid setup of the Sample monolith application and it is not fit for use in production environments.
 To learn how to configure Knot.x for use in production, see the [Production](#configuration-1) section.
+
+## Deployment options
+To deploy verticle with advanced options use following properties:
+```json
+{
+    ... 
+  "mockService": {
+    "config": {
+      "mock.data.root": "mock/service",
+      "http.port": 3000
+    },
+    "worker" : false,
```

Comment:
I think that it would be more natural to put all deployment options into `deploymentOptions` property:
```json
{
    ... 
    "mockService": {
        "mock.data.root": "mock/service",
        "http.port": 3000
    },   
    "deploymentOptions": {
      "instances": 2
    }
    ...
}
```